recipes/optick/all/conanfile.py

```diff
@@ -0,0 +1,89 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.microsoft import is_msvc_static_runtime, is_msvc
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, copy, rm, rmdir
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+import os
+
+required_conan_version = ">=2.0.0"
```

Comment:
```suggestion
required_conan_version = ">=1.54.0"
```

Please, do lock for Conan 2.x. The Conan 1.x is still mandatory for any PR: https://github.com/conan-io/conan-center-index/discussions/19104